src/xrc/xh_infobar.cpp

```diff
@@ -0,0 +1,111 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        src/xrc/xh_infobar.cpp
+// Purpose:     XML resource handler for wxInfoBar
+// Author:      Ilya Sinitsyn
+// Created:     2019/09/25
```

Comment:
Very minor, but let's use ISO format:
```suggestion
// Created:     2019-09-25
```